### PR TITLE
Unique Wither Skeleton Entity

### PR DESCRIPTION
**Pull request by hilburn**
allows it to work properly in powered spawner, plus it's really annoying

### CODE REVIEW

**Comment by tterrag1098** [comment]
Please add a config option to disable this.

File: src/main/java/crazypants/enderio/entity/SkeletonHandler.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import net.minecraft.entity.monster.EntitySkeleton;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.EntityJoinWorldEvent;
+
+public class SkeletonHandler
+{
+    @SubscribeEvent(priority = EventPriority.HIGHEST)
+    public void skeletonJoinWorld(EntityJoinWorldEvent event)
+    {
+        if (event.entity.getClass() == EntitySkeleton.class && ((EntitySkeleton)event.entity).getSkeletonType()==1)
```
Comment (tterrag1098) [inline src/main/java/crazypants/enderio/entity/SkeletonHandler.java]: No check for the config? Aren't you just spawning in an unregistered entity in that case?

**Comment by hilburn** [comment]
It checks for the config before registering the handler
https://github.com/hilburn/EnderIO/commit/77a9c0f42ac7162738155269ad6dfb9b7840ca92#diff-be8d31552bdfd3b446a181f80c4e724bL565

**Comment by tterrag1098** [comment]
Right. Derp.

**Pull request merged**